src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -436,13 +437,13 @@
                                            :unknown
 
                                            (or (= ref-prot-rest alt-prot-rest)
-                                                 (and prefer-extension-for-initial-codon-alt?
-                                                      (not= (first ref-prot-seq) (first alt-prot-seq*))))
+                                               (and prefer-extension-for-initial-codon-alt?
+                                                    (not= (first ref-prot-seq) (first alt-prot-seq*))))
                                            :extension
 
                                            :else
                                            :frame-shift)
-              (and (pos? nprefo) (= (first palt-only) \*)) :substitution
+              (and (pos? npref) (= (first palt-only) \*)) :substitution
```

Comment:
return false in a case like `pref Y` and `palt *Y`.
So I change `nprefo` to `npref`.